client/components/notice/index.jsx

```diff
@@ -109,10 +129,14 @@ export class Notice extends Component {
 			'is-dismissable': showDismiss,
 		} );
 
+		const iconName = icon || this.getIcon();
+		const iconNeedsDrop = GRIDICONS_WITH_DROP.includes( iconName );
+
 		return (
 			<div className={ classes }>
 				<span className="notice__icon-wrapper">
-					<Gridicon className="notice__icon" icon={ icon || this.getIcon() } size={ 24 } />
+					{ iconNeedsDrop && <span class="notice__icon-wrapper-drop" /> }
```

Comment:
This `class=` is throwing `Warning: Invalid DOM property `class`.`. It should be `className`

Reply:
Fixed in an upcoming push. Thanks!